frameworks/elastic/universe/marathon.json.mustache

```diff
@@ -76,6 +76,7 @@
     "MASTER_NODE_MEM": "{{master_nodes.mem}}",
     "MASTER_NODE_PLACEMENT": "{{{master_nodes.placement}}}",
     "MASTER_NODE_HEAP_MB": "{{master_nodes.heap.size}}",
+    "TASKCFG_ALL_MASTER_NODE_HEAP_MB": "{{master_nodes.heap.size}}",
```

Comment:
Is this needed? Why was the change only made for `MASTER_NODE_HEAP_MB`?

Reply:
I am using "MASTER_NODE_HEAP_MB" inside jvm.options. I was fixing it for master node but saw that all the other nodes had flag duplication error so this variable is required in all nodes to generate jvm.options.

Reply:
The corresponding variables for data, coordinator and ingest nodes are also being used in `jvm.options`, right?

Prefixing `TASKCFG_ALL` to a scheduler's environment variable causes the environment variable to be present in all pod's tasks. Do the `*_NODE_HEAP_MB` variables need it?

https://mesosphere.github.io/dcos-commons/developer-guide/#specifying-values

Reply:
As discussed, including TASKCFG_ALL for each type of node is causing all the flags appearing in each node type. So I am removing TASKCFG_ALL from moustach file. I've found an alternative  solution to this.